src/redux/features/batches/batchesAsyncThunks.ts

```diff
@@ -11,7 +11,7 @@ export type SavableBatch = (CreateBatchRequestPayload | UpdateBatchRequestPayloa
 
 export const requestSaveBatch = createAsyncThunk(
   'batches/save',
-  async (request: { batch: SavableBatch; timezone: string }, { rejectWithValue }) => {
+  async (request: { batch: SavableBatch; timezone?: string }, { rejectWithValue }) => {
```

Comment:
`timezone` is only used for new batches and defaults to UTC if `undefined`. Is that alright? Or should I add validation to ensure that `timezone` is provided if this is a "new batch" save?

Reply:
There's a timezone fallback.
For batches, timezone fallback is nursery-tz or org-tz (for display purposes)
For edit/save, fallback is nursery-tz or org-tz or user-tz.
You can use this util https://github.com/terraware/terraware-web/blob/main/src/utils/useTimeZoneUtils.ts#L84 to get the timezone for the nursery/facility

Reply:
I understand the question better now.. feels like create/update requests should be split to handle the required timezone use-case for new batches. A runtime check is probably not required. We could leave it as-is for now, splitting the func is beyond scope of this PR.